domain/src/main/kotlin/no/nav/su/se/bakover/domain/revurdering/Revurdering.kt

```diff
@@ -63,15 +63,107 @@ fun Forhåndsvarsel?.erKlarForAttestering() =
         is Forhåndsvarsel.SkalForhåndsvarsles.Besluttet -> true
     }
 
-sealed class Revurdering : Behandling, Visitable<RevurderingVisitor> {
+sealed class AbstraktRevurdering : Behandling {
```

Comment:
Stemmer heller for at vi kaller denne noe som `Endring` eller at den tar over konseptet `Revurdering` også kaller vi `Revurdering` noe mer spisset.

Reply:
Vi kan ta dette i en separat branch.